### PR TITLE
Feature/ux update

### PR DESCRIPTION
**Pull request by hermannleboss**
# Description

Update Team UX

Fixes # (issue)

## Contribution

For your PR please add a commnent to eache file edited to explain the changes you made.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

Make sur you run the following commands before submitting your PR:

- /app

```bash
npm run build
npm run test:unit
npm run type-check
npm run lint
npm run format
```

- /backend

```bash
npm run build
npm run test
npm run lint
npm run format
```

- /contract

```bash
npm run test
npm run lint
npm run format
```

- **Please check if the PR fulfills these requirements**

- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added / updated (for bug fixes / features)

### CODE REVIEW

File: app/src/components/sections/SingleTeamView/__tests__/MemberSection.spec.ts
Comment (hermannleboss) [inline app/src/components/sections/SingleTeamView/__tests__/MemberSection.spec.ts:1]: Improve the test

**Review by hermannleboss — COMMENTED** [review]
LGTM

**Review by aliassan — APPROVED** [review]
LGTM

File: app/src/components/sections/SingleTeamView/BankSection.vue
[the remark this answers is not in the document]
Comment (hermannleboss) [reply app/src/components/sections/SingleTeamView/BankSection.vue:1]: ah YES

**Pull request merged**